contribute.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review.",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "MPL2",
+    "tests": "https://circleci.com/gh/mozilla-services/mozphab/tree/master"
+  },
+  "participate": {
+    "home": "https://moz-conduit.readthedocs.io/en/latest/",
+    "docs": "https://moz-conduit.readthedocs.io/en/latest/",
+    "mailing-list": "https://www.mozilla.org/en-US/about/forums/#tools",
+    "irc": "irc://irc.mozilla.org/#phabricator",
+    "irc-contacts": [
+      "mcote",
+      "dkl",
+      "ckolos"
+    ]
+  },
+  "bugs": {
+    "list": "https://bugzilla.mozilla.org/describecomponents.cgi?product=Conduit&component=Phabricator%20Extensions#Phabricator%20Extensions",
+    "report": "https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=Phabricator%20(Upstream)",
```

Comment:
You  have the Extensions component under "list" but the Upstream component under "report". I'm not sure which is better, though... maybe "report" should just be Conduit :: General? List should be both though, i.e. https://bugzilla.mozilla.org/buglist.cgi?resolution=---&component=Phabricator%20(Upstream)&component=Phabricator%20Extensions&product=Conduit